packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -1299,6 +1299,101 @@ describe('@api RPC Server Acceptance Tests', function () {
                 const res = await relay.call('eth_call', [callData, 'latest'], requestId);
                 expect(res).to.eq(BASIC_CONTRACT_PING_RESULT);
             });
+
+            it('should fail to execute "eth_call" with correct block number', async function () {
+                const callData = {
+                    from: '0x' + accounts[2].address,
+                    to: evmAddress,
+                    data: BASIC_CONTRACT_PING_CALL_DATA
+                };
+                
+                const res = await relay.call('eth_call', [callData, '0x1'], requestId);
+                expect(res).to.eq(BASIC_CONTRACT_PING_RESULT);
+            });
+
+            it('should fail to execute "eth_call" with correct block hash', async function () {
+                const blockHash = '0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3';
+                const callData = {
+                    from: '0x' + accounts[2].address,
+                    to: evmAddress,
+                    data: BASIC_CONTRACT_PING_CALL_DATA
+                };
+                
+                const res = await relay.call('eth_call', [callData, {'blockHash' : blockHash}], requestId);
+                expect(res).to.eq(BASIC_CONTRACT_PING_RESULT);
+            });
+
+            it('should fail to execute "eth_call" with correct block hash', async function () {
```

Comment:
is this test calling with blockNumber rather than blockHash?

Reply:
Fixed.